helm/openwhisk/templates/nginx-cm.yaml

```diff
@@ -71,6 +71,7 @@ data:
         set $controllers {{ include "openwhisk.controller_host" . }};
 {{- if or (eq .Values.whisk.ingress.type "NodePort") (eq .Values.whisk.ingress.type "LoadBalancer") }}
         set $apigw {{ include "openwhisk.apigw_host" . }};
+        set $grafana {{ include "openwhisk.grafana_host" . }};
```

Comment:
Is it ok if `openwhisk.grafana_host` doesn't resolve to a host that actually exists (user metrics not enabled)?  Or should this and/or the stanza below adding `/monitoring` both be inside a 
`{{ if or .Values.metrics.prometheusEnabled .Values.metrics.userMetricsEnabled }}`
check?

Reply:
Yes, good point, a miss on my part, will check if `prometheus` or `user-metrics` is enabled.